src/main/java/sirius/biz/process/ProcessEnvironment.java

```diff
@@ -98,20 +99,19 @@ public void addDebugTiming(String counter, long millis) {
             loadPreviousTimings();
         }
 
-        return timings;
+        return Collections.unmodifiableMap(timings);
     }
 
     private void loadPreviousTimings() {
-        processes.fetchProcess(processId).ifPresent(process -> {
-            process.getPerformanceCounters().data().keySet().forEach(key -> {
-                int counter = process.getPerformanceCounters().get(key).orElse(0);
-                int timing = process.getTimings().get(key).orElse(0);
+        processes.fetchProcess(processId)
+                 .ifPresent(process -> process.getPerformanceCounters().data().keySet().forEach(key -> {
```

Comment:
please check code format settings - did maven apply the "sirius" scheme? (this might be quite alright, but normally code style shoudln't change...)

Reply:
War ne SonarLint Rule die aktiv war weil das Binding an unseren SonarQube Server nicht aktiv war

Reply:
Rules are now properly adjusted.